conf/machine/include/imx8mm-evk.inc

```diff
@@ -0,0 +1,48 @@
+MACHINEOVERRIDES =. "mx8:mx8m:mx8mm:"
+
+require conf/machine/include/imx-base.inc
+require conf/machine/include/tune-cortexa53.inc
+
+MACHINE_FEATURES += "pci wifi bluetooth optee bcm43455 bcm4356"
+MACHINE_FEATURES_append_use-nxp-bsp = " bcm4359"
+
+# NXP BSP can consume QCA9377 driver and firmware
+# Since the firmware is not available publicly, and rather distributed
+# under "Proprietary" license - we opt-out from using it in all BSPs
+# and pin it to NXP BSP only
+MACHINE_FEATURES_append_use-nxp-bsp = " qca9377"
+
+KERNEL_DEVICETREE_BASENAME ?= "${MACHINE}"
```

Comment:
Should `KERNEL_DEVICETREE_BASENAME` be left undefined in order to enforce target machine configurations to set it explicitly?

If not - then `UBOOT_CONFIG_BASENAME` should also receive default value then to maintain consistency.

Reply:
I chose to have `KERNEL_DEVICETREE_BASENAME` undefined consistently in all the .inc files.